core/conversion/converters/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+set(sub_lib_name "converters")
+
+target_sources(${lib_name} 
+    PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/NodeConverterRegistry.cpp"
+            "${CMAKE_CURRENT_SOURCE_DIR}/converter_util.cpp"
+            "${CMAKE_CURRENT_SOURCE_DIR}/Weights.cpp"
+            "${CMAKE_CURRENT_SOURCE_DIR}/impl/activation.cpp"
```

Comment:
Could we just glob the impl directory here?

Reply:
Yes we could, however globbing source files in CMake is **not** a good practice, as documented in the [CMake documentation](https://cmake.org/cmake/help/latest/command/file.html?highlight=glob#file) itself:
> Note: We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The CONFIGURE_DEPENDS flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild.

In addition, listing the files explicitly (and thus being able to change the list order) is what allowed me to recently discover the suboptimal behaviour described in #1035 . 

So I would prefer not to use glob, but obviously, if that's a blocker for you, I'll change it.

Reply:
Its fine by me as long as there is a reason, just means we will need to make sure to update both bazel and cmake as we add files